databricks_cli/click_types.py

```diff
@@ -79,3 +81,12 @@ def handle_parse_result(self, ctx, opts, args):
         if len(cleaned_opts.intersection(set(self.one_of))) > 1:
             raise UsageError('Only one of {} should be provided.'.format(self.one_of))
         return super(OneOfOption, self).handle_parse_result(ctx, opts, args)
+
+
+class ContextObject(object):
```

Comment:
Maybe we could keep this one simple by not understanding the default section name. Just store the value provided by users, or `None` otherwise.

Reply:
I've addressed your comments by doing the following. The default value for the `--profile` option is None. If it is not NONE we set the global `ContextObject`'s `._profile`. If we set it twice, then throw an exception.

Reply:
It seems `click` doesn't provide a way to tell if the value was explicitly specified by users or the default. Then the solution looks good to me.